README.md

```diff
@@ -55,8 +55,8 @@ send them timely push notifications via [FCM (Firebase Cloud Messaging)](https:/
       ```kotlin
       // build.gradle.kts
       dependencies {
-          implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:3.0.3")
-          implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:3.0.3")
+          implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:3.2.0-alpha.2")
```

Comment:
do we want to replace with actual release versions or is that a separate PR?

Reply:
We can do it in this PR, we'll have a few more last minute additions though, like version bump, readme, possibly bug fixes from our next test app build